subcommands/keys/cmd.go

```diff
@@ -27,8 +27,17 @@ to own the PKI infrastructure of their factory. This infrastructure is used
 to manage mutual TLS between your devices and the Foundries.io device gateway.`,
 }
 
+var tufCmd = &cobra.Command{
+	Use:   "tuf",
+	Short: "Manage The Update Framework Keys for your factory",
+	Long: `Every factory uses TUF to provision OTA updates to devices.
+This command allows customers to own TUF PKI of their factory in order to
```

Comment:
`PKI` term is bit confusing in this context.

Reply:
This command manages _TUF private keys_; I thought _TUF PKI_ is enough sinonymous with that. I can change it to the former if you like it more; or use another your suggestion.

Reply:
I'd leave PKI out just because its started to be used exclusively for device-gateway PKI. You also might double check elsewhere to see if we do Factory or factory in usage statements, I don't remember

Reply:
https://github.com/foundriesio/fioctl/pull/184/commits/bc1eeb2de75e2bbf2ee84951fbcae0f109d01cc4 (squash later)